### DISCUSSION TITLE
MiraiMC发布在Maven仓库的版本有误

### DISCUSSION THREAD

**Discussion by Muska-Ami** [DreamVoid/MiraiMC#146] [问答 / Q&A] [answered]
### 问题描述

如题
从图片可以看出发布的版本设置有误
这可能是pom.xml引入无效的原因
![Screenshot_20220428_234940_com microsoft emmx_edit_13287109023493](https://user-images.githubusercontent.com/79651409/165793396-0f76fded-309c-454a-bb7d-0f3924bf6c58.jpg)

### 复现方法

1. 请自行查看maven仓库https://s01.oss.sonatype.org/#nexus-search;quick~miraimc

### 补充说明

我看了一下子文件夹下的pom.xml,虽然版本号确实有变更，但是发布至maven仓库时并没有变更版本号，但是引入1.5-pre1的包确包含1.6.1的方法

- [x] 我确认我已认真阅读所有注意事项并认真填写表单。

**Comment by DreamVoid** [comment, marked as answer]
不知道你有没有发现除了`MiraiMC-Integration`都是1.5-pre1，因为在1.5我把项目改成多模块的时候还不清楚中央仓库的工作原理，误操作把一堆分散的包发上去了，结果发现删不掉。自述文件也说了用`MiraiMC-Integration`构件。
至于为什么要把5个构件整合到一个，因为我想统计[被引用情况](https://github.com/DreamVoid/MiraiMC/network/dependents?package_id=UGFja2FnZS0yNzU2ODU5NTAy)
还有，这种问题发到讨论，滥用issue以后我就直接关掉不回复了

**Reply by Muska-Ami** [reply]
你在MiraiMC-Integration里面的包的pom.xml引用的是其他零散包，那么应该继续发布而不是停止发布，这会导致Maven无法识别版本
